src/app/containers/PageHandlers/withLoading.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { bool, element } from 'prop-types';
-import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
+import { GhostGrid, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const WithLoading = Component => {
   const LoadingContainer = ({ loading, ...props }) => {
     if (!loading) return <Component {...props} />;
     return (
       <main role="main">
-        <GhostWrapper>
+        <GhostGrid>
```

Comment:
Are UX okay with the loading state having the Ghost background colour? I suppose there is unlikely to be front page to front page navigation, so a jump from white->ghost(loading state)->white will be unlikely.

Reply:
If raise it and make an issue if it is, this should be fine for v1 though :)